akka-docs/src/main/paradox/additional/monitoring.md

```diff
@@ -1,3 +1,5 @@
 # Monitoring
 
 Aside from log monitoring and the monitoring provided by your platform provider, the [Lightbend Telemetry](https://developer.lightbend.com/docs/telemetry/current/instrumentations/akka/akka.html) product (which is available through a [Lightbend Platform Subscription](https://www.lightbend.com/lightbend-platform-subscription)) can provide additional insights in the run-time characteristics of your application.
+
+[Lightbend Telemetry](https://developer.lightbend.com/docs/telemetry/current/home.html) provides distributed tracing, metrics and monitoring for Akka Actors, Cluster, HTTP and more
```

Comment:
Isn't it a bit weird with two links with the same title in the same short page going to different URLs?

Reply:
Suggestion added to include this in the first paragraph.